taichi/backends/cuda/cuda_driver.cpp

```diff
@@ -15,6 +16,8 @@ std::string get_cuda_error_message(uint32 err) {
 }
 
 bool CUDADriver::detected() {
+  if (!get_environ_config("TI_WITH_CUDA", true))
```

Comment:
Should we use `1` instead of `true` here since `get_environ_config` returns an int?